### ISSUE TITLE
Are you going to release the train code?

### ISSUE THREAD

**Issue by Li-rr** [AaronJackson/vrn#151]
Very cool paper.
Do you have any plans to release it?

**Comment by AaronJackson** [comment]
Just going through some old / open issues.
I thought about it but it has become too much of a hassle to pull all the parts together sorry. The training code itself is very simple, essentially just a 2D segmentation network with a Binary Cross Entropy loss. Voxelising the dataset and making sure it is all aligned is the hard (and computationally expensive) part.
